polldaddy.php

```diff
@@ -782,34 +782,40 @@ function management_page_load() {
 					$this->errors->add( 'answer', __( 'Invalid answers', 'polldaddy' ) );
 
 				$answers = array();
-				foreach ( $_POST['answer'] as $answer_id => $answer ) {
-					$answer = stripslashes( trim( $answer ) );
+				if ( ! empty( $_POST['answer'] ) ) {
```

Comment:
Maybe should we add the same `if` to the `create-poll` case?

Reply:
Good catch! a490f4a24df6d4ae20dbec7e8cd5b895e36dd37e